aviatrix/data_source_aviatrix_firewall_instance_images.go

```diff
@@ -73,3 +84,115 @@ func dataSourceAviatrixFirewallInstanceImagesRead(d *schema.ResourceData, meta i
 	d.SetId(vpcId)
 	return nil
 }
+
```

Comment:
Everything under here can probably me moved to `aviatrix/utils.go`

Reply:
yes, move all sort methods into utils.go file, thank you